ncm-useraccess/src/main/perl/useraccess.pod

```diff
@@ -163,10 +153,7 @@ may consume lots of disk space.
 
 =head1 EXAMPLES
 
-You still have questions? I hope the following examples will make it
-clearer.
-
-=head2 One user who can be sued
+=head2 One user who can be sued:
 
 This example illustrates how tricky ACLs can get:
```

Comment:
While you are at that, remove this section.  PAM ACLs should be avoided, and the only user has long left the community.

Or, at least, put a bold text mentioning that this feature should be avoided at all costs.